test/test_prototype_models.py

```diff
@@ -83,6 +83,38 @@ def test_naming_conventions(model_fn):
     assert len(weights_enum) == 0 or hasattr(weights_enum, "default")
 
 
+@pytest.mark.parametrize(
+    "model_fn",
+    TM.get_models_from_module(models)
+    + TM.get_models_from_module(models.detection)
+    + TM.get_models_from_module(models.quantization)
+    + TM.get_models_from_module(models.segmentation)
+    + TM.get_models_from_module(models.video),
+)
+def test_schema_meta_validation(model_fn):
+    classification_fields = ["size", "categories", "acc@1", "acc@5"]
+    defaults = {
+        "all": ["interpolation", "recipe"],
```

Comment:
Though currently the `categories` field is used in all existing models, I haven't put it in the `all` key because of the upcoming addition of optical flow models.

Reply:
Q: Is the interpolation field related to the resize transform?

Reply:
That's correct. It affects resize and affine transforms. You could lose significant accuracy if you use a different interpolation on inference than the one used on training. That's why we put it on the meta-data.